packages/jsii-pacmak/lib/targets/version-utils.ts

```diff
@@ -55,10 +67,90 @@ export function toPythonVersionRange(semverRange: string): string {
     .join(', ');
 }
 
+/**
+ * Converts an original version number from the NPM convention to the target
+ * language's convention for expressing the same. For versions that do not
+ * include a prerelease identifier, this always returns the assembly version
+ * unmodified.
+ *
+ * @param assemblyVersion the assembly version being released
+ * @param target          the target language for which the verison is destined
+ *
+ * @returns the version that should be serialized
+ */
+export function toReleaseVersion(
+  assemblyVersion: string,
+  target: TargetName,
+): string {
+  const version = parse(assemblyVersion, { includePrerelease: true });
+  if (version == null) {
+    throw new Error(
+      `Unable to parse the provided assembly version: "${assemblyVersion}"`,
+    );
+  }
+  if (version.prerelease.length === 0) {
+    return assemblyVersion;
+  }
+  switch (target) {
+    case 'python':
```

Comment:
Hmmm. Do we not have an enum or something that represents the list of supported target langs?

Reply:
Not an enum; but the `TargetName` is a string-literal-union type, so one can't make typos here without the compiler noticing.